test/blackbox-tests/test-cases/external-lib-deps/fix-bug.t/run.t

```diff
@@ -0,0 +1,8 @@
+Expected: The dependency "inter_lib" in ./dune file was printed, the "inter_lib" lib is already
```

Comment:
Can you rename `fix-bug.t` to something a bit more descriptive?

Reply:
like `exclude-internal-deps.t` is it better ?

Reply:
yes, that works